src/elements/content-sidebar/versions/VersionsItemButton.js

```diff
@@ -32,51 +39,43 @@ class VersionsItemButton extends React.Component<Props> {
         }
     }
 
-    setButtonRef = (buttonRef: ?HTMLDivElement): void => {
-        this.buttonRef = buttonRef;
-    };
-
     setScroll = () => {
         const { isSelected } = this.props;
+        const { current: buttonRef } = this.buttonRef;
 
-        if (this.buttonRef && isSelected) {
-            scrollIntoView(this.buttonRef);
+        if (buttonRef && isSelected) {
+            scrollIntoView(buttonRef);
         }
     };
 
     render() {
-        const { children, className, isDisabled, isSelected, onActivate } = this.props;
-        const buttonClassName = classNames(className, {
+        const { children, fileId, isCurrent, isDisabled, isSelected, onClick } = this.props;
+        const buttonClassName = classNames('bcs-VersionsItemButton', {
             'bcs-is-disabled': isDisabled,
             'bcs-is-selected': isSelected,
         });
 
         return (
-            <div
+            <button
                 aria-disabled={isDisabled}
                 className={buttonClassName}
-                data-resin-target="versions-item-button"
+                data-resin-iscurrent={isCurrent}
+                data-resin-itemid={fileId}
+                data-resin-target="select"
                 data-testid="versions-item-button"
                 onClick={event => {
-                    if (isLeftClick(event)) {
-                        onActivate(event);
-                    }
-                }}
-                onKeyPress={event => {
-                    if (isActivateKey(event)) {
-                        if (event.key === KEYS.space) {
-                            event.preventDefault(); // Prevent scroll on space key press
-                        }
-
-                        onActivate(event);
+                    if (isDisabled) {
```

Comment:
Could we set the disabled property of the button to avoid manually preventing the click event?

Reply:
This is how `PlainButton` handles disabled behavior, so I'm just aping it here. I don't want to use `PlainButton` because it messes with the styles really badly in this case.